### PR TITLE
Fixed share button bug

### LINKED ISSUE #540: Fix Profile Share button for some users
location - logged in user profile
the close button goes behind the facebook icon
image (1)

### PR DESCRIPTION
**Pull request by rishsingh07**
Fixed #540 
@norbertschuler  Please check.

### CODE REVIEW

**Comment by norbertschuler** [comment]
Still wrong testing this PR with https://planet-webapp-ltv8f0s1t-planetapp.vercel.app/t/norbert-schuler:

Bildschirmfoto 2021-02-25 um 10 43 37

@sarveshpro can you test with Chrome?

**Comment by harshvitra** [comment]
@rishsingh07 Can you implement the same changes for Personal profiles. 
(Log in and check from your own profile page)

**Comment by harshvitra** [comment]
> Still wrong testing this PR with https://planet-webapp-ltv8f0s1t-planetapp.vercel.app/t/norbert-schuler:

There are 2 different menus: 1 for logged in users and 1 when I check someone's profile. 

Rishabh fixed one and the other has to be fixed

**Review by norbertschuler — APPROVED** [review]
LGFM

File: src/features/user/UserProfile/components/UserProfileOptions.tsx [hunk 1]
```diff
@@ -148,7 +148,8 @@ export default function UserProfileOptions({
           animate={{
             position: 'absolute',
             top: '35px',
-            right: divWidth > 291 ? '290px' : '-180px',
+            right: divWidth > 320 ? '290px' : '-180px',
+            backgroundColor: "red",
```
Comment (harshvitra) [inline src/features/user/UserProfile/components/UserProfileOptions.tsx:152]: Remove this

**Review by harshvitra — CHANGES_REQUESTED** [review]

**Review by harshvitra — APPROVED** [review]

**Pull request merged**